part-spec/power/pmic.json

```diff
@@ -1,71 +1,25 @@
 {
     "$id": "https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/power/pmic.json",
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "specification of pmic",
+    "title": "specification of a pmic",
     "pmic": {
         "type": "object",
+        "required": "partType",
         "properties": {
-            "linearRegulatorCount": {
-                "description": "number of linear regulators in the device",
-                "type": "number"
-            },
-            "buckRegulatorCount": {
-                "description": "number of buck regulators in the device",
-                "type": "number"
-            },
-            "boostRegulatorCount": {
-                "description": "number of boost regulators in the device",
-                "type": "number"
-            },
-            "buckBoostRegulatorCount": {
-                "description": "number of buck-boost regulators in the device",
-                "type": "number"
-            },
-            "adcCount": {
-                "description": "number of analog to digital converters in the device",
-                "type": "number"
-            },
-            "externalClockCount": {
-                "description": "number of external clocks the device requires",
-                "type": "number"
-            },
-            "internalClockCount": {
-                "description": "number of clocks/oscillators in the device",
-                "type": "number"
-            },
-            "loadSwitchCount": {
-                "description": "number of load switches in the device",
-                "type": "number"
-            },
-            "usbSwitchCount": {
-                "description": "number of USB switches in the device",
-                "type": "number"
-            },
-            "componentList": {
-                "description": "list of components integrated in the device",
-                "comment": "list should include components with an existing digital datasheet specification",
-                "examples": [
-                    "load switch",
-                    "linear regulator",
-                    "switching regulator"
+            "partType": {
+                "description": "part type",
+                "enum": [
+                    "pmic"
                 ],
-                "type": "array",
-                "items": {
-                    "type": "string"
-                }
+                "type": "string"
             },
-            "instances": {
-                "description": "definition (specification) of each instance of a component in the device",
+            "instanceSpec": {
+                "description": "array of specifications of instances in the pmic ",
                 "type": "array",
                 "items": {
-                    "$ref": "#/$defs/instances"
+                    "$ref": "../common/instanceSpec.json#/instanceSpec"
                 }
             },
-            "vin": {
```

Comment:
Since this is a PMIC, each integrated LDO/DC-DC could have a different Vin range.  So in this case, I felt that the pins description would capture this information better.